pkg/dispatcher/dispatcher_test.go

```diff
@@ -89,7 +90,7 @@ func TestMain(m *testing.M) {
 func Test_getListeners(t *testing.T) {
 	d := setUp()
 	listeners := d.GetListeners()
-	assert.Assert(t, len(listeners) == 4, "Expected to find 4 listeners in list. Got %d", len(listeners))
+	assert.Assert(t, len(listeners) == 7, "Expected to find 4 listeners in list. Got %d", len(listeners))
```

Comment:
change to Assert.Equal, and you can remove the comment

Reply:
done

Reply:
the message still says **4**

Reply:
done